src/components/Admin/index.jsx

```diff
@@ -199,6 +226,42 @@ class Admin extends React.Component {
     return [courseCompletions, enrolledLearners, numberOfUsers].every(item => item === 0);
   }
 
+  renderButtonsRow() {
+    if (this.displaySearchBar()) {
+      return (
+        <div className="col-12 col-md-6 text-md-right">
+          <div className="row">
+            <div className="col-sm-12 col-md-7 pr-0">
```

Comment:
Could we use the `.no-gutters` on the `.row` instead of the padding utility classes?